pkg/common/oras/authprovider/authprovider.go

```diff
@@ -60,6 +61,7 @@ type defaultAuthProviderConf struct {
 }
 
 const DefaultAuthProviderName string = "dockerConfig"
+const DefaultDockerAuthTTL = 1 * time.Hour
```

Comment:
This is an entirely arbitrary ttl. Docker config doesn't seem to have a concept of TTL for the creds stored. Should we adjust this time?

Reply:
What are typical ttl for other authprovider tokens?

Reply:
ACR has a ttl of 1 hour for its access token and 3 hours for its refresh token. That was mainly my inspiration for the 1 hour ttl as the default. I'm not familiar with other registries. I will take a look at other projects to see if there's some standard adopted.

Reply:
The other option we have is to make the ttl 0 which in turn keeps value cached forever unless forcefully evicted during an authentication 401 error.

Reply:
If this is similar to ACR then its fair to leave it at 1 hour.  We can adjust later if we need to.

Reply:
wondering how does this file get updated on the cluster? i think 1hr is a good to start with if customer might manually update this file? we might want to document this somewhere?

Reply:
keeping 1 hr for now